Helper/ConfigHelper.php

```diff
@@ -1483,6 +1498,8 @@ public function showSuggestionsOnNoResultsPage($storeId = null)
     }
 
     /**
+     * NOTE: This method is currently only used in integration tests and was removed from the general implementation
```

Comment:
Out of curiosity, when was it removed from the general implementation ? :)

Reply:
I tracked it down here: https://github.com/algolia/algoliasearch-magento-2/pull/1376/files 

I was initially looking at it because it also harvests all possible currencies but realized it wasn't used anymore. My main concern is that by removing it do we expose "hidden" product attributes in certain scenarios. Need to do more investigation here.

Reply:
Oh I see, it was only used to generate secured api key... 🤔